readme.md

```diff
@@ -12,11 +12,33 @@ You will need [Composer](https://getcomposer.org/) for the Laravel Dependency. O
 $ composer install
 ```
 
-Next create a new mysql database for the randomizer (see mysql documentation for how to do this)
+Next create a new mysql database for the randomizer (see mysql documentation for how to do this, you'll need to install mysql server if it's not installed already)
 
-Modify the config/database.php with appropriate username, password, and database name in the mysql section.
+Run the following command to create a new config for the app
+```
+$ cp .env.example .env
+```
+
+Then modify .env with appropriate username, password, and database name. Change the db connection to mysql
+Example:
+```
+DB_CONNECTION=mysql
```

Comment:
I really wanted to use sqlite, that way you don't need to install mysql, but it doesn't work. Didn't feel like working out why it couldn't...

Reply:
let me see if I can work out why sqlite is having problems on a fresh clone